asf_core_data/getters/mcs_getters/get_mcs_installations.py

```diff
@@ -127,9 +144,72 @@ def get_processed_installations_data(
     """
 
     return load_s3_data(
-        base_config.BUCKET_NAME,
+        bucket_name,
         os.path.join(
             base_config.MCS_PROCESSED_FILES_PATH,
             processed_installations_file_name,
         ),
     )
+
+
+def find_most_recent_mcs_installations_batch(epc_version: str = "none") -> str:
+    """
+    Finds most recent batch of MCS installations data, joined with EPC data in specified format if specified.
+    Args:
+        epc_version: One of "none", "full", or "most_relevant".
+            - "none" returns just installation data
+            - "full" returns installation data with each property's entire EPC history attached
+            - "most_relevant" selects the most recent EPC from before the HP installation if one exists or the earliest
+            EPC from after the HP installation otherwise. Defaults to "none".
+    Returns:
+        str: Filename of most recent MCS installations (+ EPC) data.
+    """
+    bucket = s3.Bucket(bucket_name)
+    file_list = [
+        object.key for object in bucket.objects.filter(Prefix=mcs_processed_dir)
+    ]
+    file_prefix = keyword_to_path_dict[epc_version].split("{")[0]
+    matches = [
+        filename
+        for filename in file_list
+        if ("/" + re.split("[0-9]", filename)[0] == file_prefix)
+    ]
+    try:
+        latest_version = max(matches)
+        print(f"Latest version available on S3: <{latest_version}>")
```

Comment:
nice-to-have (logging)
```suggestion
        logger.info(f"Latest version available on S3: <{latest_version}>")
```